frontend/src/components/mine/Compliance/MineComplianceInfo.js

```diff
@@ -140,7 +133,9 @@ export class MineComplianceInfo extends Component {
                   <Col span={4}>
                     <div className="center">
                       <p className="info-display">
-                        {formatDate(this.props.mineComplianceInfo.last_inspection) || "-"}
+                        {this.props.mineComplianceInfo.last_inspection === null
+                          ? String.NO_NRIS_INSPECTIONS
+                          : formatDate(this.props.mineComplianceInfo.last_inspection)}
```

Comment:
Meredith and I recently updated the `formatDate` function to handle falsey values so that it cane be used like this:
```javascript
formatDate(this.props.mineComplianceInfo.last_inspection) || String.NO_NRIS_INSPECTIONS
```

The idea was to move all the repeated null checks into the function because they were being written over and over

Reply:
Thanks for this, has been updated 👍